torchaudio/datasets/vctk.py

```diff
@@ -252,6 +272,14 @@ def _load_sample(self, speaker_id: str, utterance_id: str, mic_id: str) -> Sampl
         return Sample(waveform, sample_rate, utterance, speaker_id, utterance_id)
 
     def __getitem__(self, n: int) -> Sample:
+        """Load the n-th sample from the dataset.
```

Comment:
Interesting that the tuple description is kept on the class docstring for this dataset.

Reply:
Oh this is because we intend to change NamedTuple to a normal tuple for consistency reason before the release. (so very soon) So this way I do not have to change the docstrings twice.

Reply:
oh I miss red your comment. it's an oversight. I will remove it from there.